### PR TITLE
Database ready

### PR DESCRIPTION
**Pull request by johndlopez**

### CODE REVIEW

File: db/db.js [hunk 1]
```diff
@@ -28,37 +28,39 @@ const songSchema = mongoose.Schema({
...
     } else {
-      console.log('Import successful', data);
+      console.log('Song found!', data);
+      callback(null, data[0]);
     }
   });
 }
 
-Song.find((error, songs) => {
+const getAll = (callback) => {
```
Comment (kristiedesiree) [inline db/db.js]: What role is the callback playing here? I don't see it being invoked

File: db/db.js [hunk 2]
```diff
@@ -28,37 +28,39 @@ const songSchema = mongoose.Schema({
...
+  Song.find({ id: 15 }).exec((error, data) => {
```
Comment (kristiedesiree) [inline db/db.js]: Is there a reason why the ID is set to 15, or is that just a placeholder for now?

**Review by kristiedesiree — APPROVED** [review]

File: db/db.js (hunk 2 above)
Comment (johndlopez) [reply db/db.js]: Oh that is a mistake, I'll change it

File: db/db.js (hunk 1 above)
Comment (johndlopez) [reply db/db.js]: Ahh this is leftover from testing.

**Pull request merged**